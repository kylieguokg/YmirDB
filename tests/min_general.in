set a 0
get a
set b a 19
get b
min a
min b
set c -1000
get c
append b c
get b
min a
min b
min c
set d -2000
get d
push c d
get c
LIST ENTRIES
min a
min b
min c
min d
bye
