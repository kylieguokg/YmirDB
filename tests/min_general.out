> ok

> [0]

> ok

> [a 19]

> 0

> 0

> ok

> [-1000]

> ok

> [a 19 c]

> 0

> -1000

> -1000

> ok

> [-2000]

> ok

> [d -1000]

> d [-2000]
c [d -1000]
b [a 19 c]
a [0]

> 0

> -2000

> -2000

> -2000

> bye
